packages/mermaid/src/docs/.vitepress/scripts/fetch-avatars.ts

```diff
@@ -32,7 +36,7 @@ async function fetchAvatars() {
     download(`https://github.com/${name}.png?size=100`, getAvatarPath(name));
```

Comment:
I think we should probably modify the `download()` function so that it's doesn't run fully in parallel. Every time I try to download the avatars, GitHub seems to block my computer from accessing GitHub for a few minutes :laughing:

We should also set the appropriate HTTP headers for GitHub's API.

We could also install a JavaScript client to download the avatar, which should handle rate-limiting, headers, auth, etc. all for us. The only problem is that there doesn't seem to be a REST API JavaScript client that can download the avatar. Only the GraphQL API has an `avatarUrl` field: https://docs.github.com/en/graphql/reference/objects#user

Reply:
It was, I was tired of waiting and replaced it with parallel version. We need either to satisfy GH rate limit, or may be refuse from downloading avatars in local dev at all, or limit them like to 10 only, that would be enough for debugging and visual purposes.

Reply:
At least in CI, there's a [`GITHUB_TOKEN` environment variable](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret) that can be used with the API to heavily reduce rate limits. :)

I'd be :+1: to just downloading the first 10 pictures, then replacing all the other avatars with a dummy picture, like 
![image](https://github.com/mermaid-js/mermaid/assets/19716675/0aca87be-4bc7-4e30-bdf4-c120159b6637)